src/model/pitch.rs

```diff
@@ -24,6 +24,25 @@ impl Pitch {
     pub fn all() -> impl Iterator<Item = Pitch> {
         Letter::iter().flat_map(|l| Modifier::iter().map(move |m| Pitch::new(l, m)))
     }
+
+    pub fn index(&self) -> usize {
```

Comment:
This is related to the thread re different pitches vs aliases, this seems like a little bit of an "inversion" vs what I was describing but seems fine for now?